src/seedu/addressbook/AddressBook.java

```diff
@@ -225,13 +216,22 @@ public static void main(String[] args) {
      * signature anyway.
      * ====================================================================
      */
+    
+    private static void processUserCommand() {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.